diff --git a/node_modules/jwks-rsa/lib/integrations/express.js b/node_modules/jwks-rsa/lib/integrations/express.js
index 6cc77c5..98c8027 100644
--- a/node_modules/jwks-rsa/lib/integrations/express.js
+++ b/node_modules/jwks-rsa/lib/integrations/express.js
@@ -24,7 +24,7 @@ module.exports.expressJwtSecret = function (options) {
   var client = new _JwksClient.JwksClient(options);
   var onError = options.handleSigningKeyError || handleSigningKeyError;
 
-  return function secretProvider(req, header, payload, cb) {
+  return function secretProvider(header, cb) {
     // Only RS256 is supported.
     if (!header || header.alg !== 'RS256') {
       return cb(null, null);
